.clang-format

```diff
@@ -0,0 +1,100 @@
+# Do not allow short blocks (e.g., if statements without braces) on a single line.
+AllowShortBlocksOnASingleLine: Never
+# Do not allow short case labels in switch statements on a single line.
+AllowShortCaseLabelsOnASingleLine: false
+# Do not allow short enums on a single line.
+AllowShortEnumsOnASingleLine: false
+# Only allow empty functions on a single line.
+AllowShortFunctionsOnASingleLine: Empty
+# Do not allow short if statements on a single line.
+AllowShortIfStatementsOnASingleLine: Never
+# Do not allow short loops on a single line.
+AllowShortLoopsOnASingleLine: false
+# Add a space after the colon in bit fields.
+BitFieldColonSpacing: After
+# Break before binary operators that are not assignments.
+# E.g. break before "+", but not before "+=".
+BreakBeforeBinaryOperators: NonAssignment
+# Attach opening braces to the same line as the statement or declaration it belongs to.
+BreakBeforeBraces: Attach
+# Ternary operators will be placed after line breaks.
+BreakBeforeTernaryOperators: true
+# Break string literals if they are too long.
+BreakStringLiterals: true
+# Indentation width for constructor initializers.
+ConstructorInitializerIndentWidth: 2
+# Indentation width for continuation lines.
+ContinuationIndentWidth: 2
+# Format braced lists as best suited for C++11 braced lists.
+Cpp11BracedListStyle: true
+# Use a fixed alignment for all pointers.
+DerivePointerAlignment: false
+# Do not indent case blocks in switch statements.
+IndentCaseBlocks: false
+# Indent case labels in switch statements.
+IndentCaseLabels: true
+# Standard indentation width.
+IndentWidth: 2
+# Do not indent if a function definition or declaration is wrapped after the type.
+IndentWrappedFunctionNames: false
+# Insert a newline at end of file if missing.
+InsertNewlineAtEOF: true
+# Maximum number of consecutive empty lines to keep.
+MaxEmptyLinesToKeep: 2
+# Do not indent namespaces.
+NamespaceIndentation: None
+# Penalty for indentation using spaces rather than tabs.
+PenaltyIndentedWhitespace: 0
+# Pointer alignment (Left or Right).
+PointerAlignment: Left
+# Separate definition blocks (Always, Never, or Leave).
+SeparateDefinitionBlocks: Always
+# Number of spaces before trailing comments.
+SpacesBeforeTrailingComments: 1
+# Do not add a space after a C-style cast.
+SpaceAfterCStyleCast: false
+# Do not add a space after the template keyword.
+SpaceAfterTemplateKeyword: false
+# Sort includes (CaseSensitive, CaseInsensitive, or None).
+SortIncludes: CaseInsensitive
+# C++ standard to be followed (e.g., c++98, c++11, c++14, c++17, c++20).
+Standard: c++20
+# Add a space before opening parentheses in control statements.
+SpaceBeforeParens: ControlStatements
+# Do not add a space before square brackets.
+SpaceBeforeSquareBrackets: false
+# Width of a tab character.
+TabWidth: 2
+# Always use tabs for indentation.
+UseTab: Always
+# Don't touch lines used for translation.
+WhitespaceSensitiveMacros: [
+	'translate_plural',
+	'translate'
```

Comment:
translate_ctxt should probably be added here too. Maybe I'll fix this properly one day and we can get rid of this hack. :)